Missionframework/modules/26_cratefiller/ui/KPLIB_cratefiller.hpp

```diff
@@ -182,13 +182,13 @@ class KPLIB_cratefiller {
         class KPLIB_LeftEquipmentListButton: KPGUI_PRE_BUTTON {
             idc = KPLIB_IDC_CRATEFILLER_LEFTEQUIPMENTBUTTON;
             text = "-";
-            onButtonClick = "[68745] call KPLIB_fnc_cratefiller_removeEquipment";
```

Comment:
The control ID won't be passed correct when using `KPLIB_IDC_CRATEFILLER_EQUIPMENTLIST` as function argument.

Reply:
It should be if you would use single quotes.

Reply:
Doesn't work with single quotes.
`onButtonClick = '[KPLIB_IDC_CRATEFILLER_INVENTORYLIST] call KPLIB_fnc_cratefiller_removeEquipment';`
Passed param is 0 (or delivered as default param value)